apps/server/src/api-integration/integration.controller.ts

```diff
@@ -43,22 +48,33 @@ const actionHandlers: Record<string, ActionHandler> = {
     const data = payload[id as keyof typeof payload];
     const patchEvent: Partial<OntimeEvent> & { id: string } = { id };
 
+    let throttleProperty = false;
+
     Object.entries(data).forEach(([property, value]) => {
       if (typeof property !== 'string' || value === undefined) {
         throw new Error('Invalid property or value');
       }
 
       const newObjectProperty = parseProperty(property, value);
 
+      if (!throttleProperty && !propertiesWhitelist.includes(Object.keys(newObjectProperty)[0])) {
```

Comment:
Few suggestions:
1. I think the logic for checking if a property is not whitelisted should be migrated to the scope of in rundownCacheUtils.

2. I also wonder if we could simplify with simplify the throttle assignment to simply 
`throttleProperty = throttleProperty || notWhitelisted(property)`

3. the user will always see their change (eventually) in that case, does the throttled message achieve anything?

Reply:
i seams that `isDataStale` is doing what we want but the name dose not imply that 
should we maybe rename it?

Reply:
> does the throttled message achieve anything

it's very helpful for testing, I could remove it now and return `undefined` so that the ws connation dosn't generate a reply

Reply:
i think isDataStale is checking two objects against each other. in our case it would mean we iterate through the patch object twice.

Since we are already iterating, it may be simpler to extract the code that checks if a key is in the enum

Reply:
https://github.com/cpvalente/ontime/blob/8d526c7d31bd3bb765a633ba6030638b43758617/apps/server/src/services/rundown-service/rundownCacheUtils.ts#L121-L123